Makefile

```diff
@@ -157,32 +157,27 @@ all: binaries
 android-dist: genconfig android
 
 # This is to be called within the docker image.
-docker-genassets: require-npm
+genassets: require-npm
```

Comment:
I think that would probably be wise @xiam, as we also want to push out valencia ASAP so we can get back to pro, so we probably want to keep all pieces as stable as possible (i.e. this isn't on the critical path for release).

I would argue we ideally would have our dev environments be as close as possible to production to essentially get free QA, but I don't feel strongly about docker versus local dev environment at first glance.